ql/src/java/org/apache/hadoop/hive/ql/session/ProcessListInfo.java

```diff
@@ -32,10 +32,11 @@ public class ProcessListInfo {
   private final String runtime;  // tracks only running portion of the query.
   private final long elapsedTime;
   private final String state;
+  private final long txnId;
```

Comment:
I might be wrong. Don't we need to read this variable in [ShowProcessListProcessor.java](https://github.com/apache/hive/blob/master/ql/src/java/org/apache/hadoop/hive/ql/processors/ShowProcessListProcessor.java)?